README.md

```diff
@@ -31,6 +31,7 @@ I'm looking for a full remote position (EU)! See [edoardottt.com/cv](https://www
 | Auth0 | All | Full remote okay | [Link](https://auth0.com/careers) | ? |
 | bill.com |  | Some areas | [Link](https://www.bill.com/about-us/jobs) | ? |
 | Bishop Fox |  | Some areas | [Link](https://www.bishopfox.com/jobs/) | ? |
+| Bitdefender | Some | Full remote (Regional) | [Link](https://www.bitdefender.com/company/job-opportunities/) | ? |
```

Comment:
They currently have at least [one remote security position in the U.S.](https://bitdefender.csod.com/ux/ats/careersite/1/home/requisition/401?c=bitdefender), I believe I've seen some remote security positions in Romania listed on their site in the past...